### PR TITLE
Issue #13672: Kill mutation in Common Profile

### PR DESCRIPTION
**Pull request by Kevin222004**
Issue #13672: Kill mutation in Common Profile

# Explaination
Test

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java [hunk 1]
```diff
@@ -46,7 +46,7 @@
 /**
  * Unit test for ConfigurationLoader.
  */
-public class ConfigurationLoaderTest extends AbstractPathTestSupport {
+public class ConfigurationLoaderTest extends AbstractModuleTestSupport {
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java:49]: Why this change is required?

**Review by romani — CHANGES_REQUESTED** [review]
Items:

**Review by romani — APPROVED** [review]
Ok to merge if ci pass

File: src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java (hunk 1 above)
Comment (Kevin222004) [reply src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java]: Done
